examples/Language_Translation/English_to_French_Translation/create_mar.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+rm -rf fairseq
+rm wmt14.en-fr.joined-dict.transformer.tar.bz2
+rm -rf fairseq-build
+
+#installing cython
+pip install Cython
+
+#download zip file of fairseq
+git clone https://github.com/pytorch/fairseq
+cd fairseq/
+python3 setup.py sdist bdist_wheel
+cd ..
+mkdir fairseq-build
+cp fairseq/dist/fairseq-0.9.0.tar.gz fairseq-build/
```

Comment:
@AshwinChafale For ease of maintainability with future releases of Fairseq, it will be better to not peg to a specific version. Is there any special reason for using the 0.9.0 version explicitly?

Reply:
"python3 setup.py sdist bdist_wheel" command has auto-generated fairseq-0.9.0.tar.gz in "/dist/" folder. There is no specific reason to use 0.9.0 version.
Furthermore, I have also updated the "create_mar.sh" to use the latest build file rather than using a specific version